airbyte_cdk/sources/file_based/config/abstract_file_based_spec.py

```diff
@@ -31,6 +31,12 @@ class Config(OneOfOptionConfig):
 
     delivery_type: Literal["use_file_transfer"] = Field("use_file_transfer", const=True)
 
+    preserve_subdirectories_directories: bool = Field(
+        title="Preserve Sub-Directories in File Paths",
+        description="If enabled replicate source folder structure",
```

Comment:
Wdyt of expanding the description (and tooltip help) a bit?

```suggestion
        description=(
            "If enabled, sends subdirectory folder structure "
            "along with source file names to the destination. "
            "Otherwise, files will be synced by their names only. "
            "This option is ignored when file-based replication is not enabled." 
        ),
```

Reply:
Great, just applied this messaging, thanks.